disco/disco.go

```diff
@@ -42,6 +41,7 @@ var httpTransport = defaultHttpTransport()
 // hostnames and caches the results by hostname to avoid repeated requests
 // for the same information.
 type Disco struct {
+	aliasMap  map[svchost.Hostname]svchost.Hostname
```

Comment:
little nit, not super important: the `Map` suffix here seems redundant since this field is clearly of a map type.